src/runtime/components/forms/Form.vue

```diff
@@ -218,6 +225,18 @@ function isZodSchema (schema: any): schema is ZodSchema {
   return schema.parse !== undefined
 }
 
+async function getSuperStructErrors (state: any, schema: Struct<any, any>): Promise<FormError[]> {
+  const [err] = validateSuperStruct(state, schema)
```

Comment:
You can try this to avoid importing the validate method from `superstruct`
```suggestion
  const [err] = schema.validate(state)
```